src/plugins/onePingPerUser/index.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Vencord, a Discord client mod
+ * Copyright (c) 2023 Vendicated and contributors
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+import { ChannelStore, ReadStateStore } from "@webpack/common";
+import { Message } from "discord-types/general";
+
+enum ChannelType {
+    DM = 1,
+    GROUP_DM = 3
+}
+
+export default definePlugin({
+    name: "OnePingPerUser",
+    description: "Limits the ping audio notification from playing more than once per user until messages are marked as read to prevent audio spam and hearing multiple notifications as a cue that more than one user may have messaged you",
```

Comment:
this sentence is veeery long, maybe add some puncutation

Reply:
maybe something like this would be better?
```suggestion
    description: "Only play the message notification sound once per user, no matter how many messages they send. Reading their messages resets the limit",
```